src/boot/efi/measure.c

```diff
@@ -42,6 +42,51 @@ static EFI_STATUS tpm1_measure_to_pcr_and_event_log(
                         &event_log_last);
 }
 
+static EFI_STATUS tpm2_measure_to_pcr_and_tagged_event_log(
+                EFI_TCG2_PROTOCOL *tcg,
+                uint32_t pcrindex,
+                EFI_PHYSICAL_ADDRESS buffer,
+                uint64_t buffer_size,
+                uint32_t event_id,
+                const char16_t *description) {
+
+        _cleanup_free_ struct event {
+                EFI_TCG2_EVENT tcg_event;
+                EFI_TCG2_TAGGED_EVENT tcg_tagged_event;
+        } *event;
```

Comment:
This should really be NULLed, or better yet, move the definition to the first use. It's not the 80s anymore.